libs/media-player/src/lib/video-player.tsx

```diff
@@ -12,23 +12,36 @@ const isVideoMIMEType = (input: unknown): input is VideoMIMEType =>
 interface VideoPlayerProps {
     videoUrl: string;
     mimeType?: VideoMIMEType;
+    index?: unknown;
 }
 
 const StyledVideoPlayer = styled('video')`
     border-radius: 5px;
     width: 100%;
 `;
 
-export const VideoPlayer = ({ videoUrl, mimeType }: VideoPlayerProps) => {
+export const VideoPlayer = ({ videoUrl, mimeType, index }: VideoPlayerProps) => {
     return (
         <StyledVideoPlayer controls>
             {isVideoMIMEType(mimeType) ? (
-                <source style={{ padding: 0 }} src={videoUrl} type={mimeType} />
+                <source
+                    data-testid="video-player"
+                    style={{ padding: 0 }}
+                    src={videoUrl}
+                    key={`${videoUrl}-${index}`}
```

Comment:
Why do we need the index here?  Is there a case where the same video url would display multiple times on the page.  Sorry, I'm just missing something.

Reply:
I agree with @yaanahuu2 on this one. I do not want to see `index` as the video player props. It is not the video player's responsibility to know it's place in a parents list or even know whether it appears in a list or in isolation.

Reply:
If you are adding multiple sources, use `mimeType` as the key.

Reply:
Done.